docs/codacy-api/examples/identifying-commits-without-coverage-data.md

```diff
@@ -0,0 +1,73 @@
+---
+description: Example of how to identify commits missing the coverage data required for calculating the coverage metrics of pull requests.
+---
+
+# Identifying commits without coverage data
+
+To calculate the supported coverage metrics for pull requests, Codacy requires that at least the following commits provide coverage data:
+
+-   The common ancestor commit of the pull request branch and the target branch
+-   The head commit of the pull request branch
+
+The following diagram highlights the commits that must have received coverage data for Codacy to display the coverage variation metric on a pull request:
+
+![Commits that must have received coverage data](../../coverage-reporter/images/coverage-pr-commits.png)
+
+However, different factors may prevent your setup from correctly reporting coverage data for the required commits.
```

Comment:
If I understand correctly, these different factors are all in the hands of the customer. That is, **they** must set everything up correctly.

The way it's phrased, a user may read "different factor" as potentially something internal to Codacy (at least I did). I'd rephrase it to clarify that Codacy is at the receiving end and expected to work. Something like:

```suggestion
However, a misconfigured setup may fail to report coverage data to Codacy for the required commits.
```

Reply:
If you [check the troubleshooting table](https://docs.codacy.com/coverage-reporter/#status-commit-not-found) for the possible error status (which is not exhaustive) you'll probably agree that not every scenario is under the user's control. So I feel more comfortable with leaving the more ambiguous wording here.